addons/point_of_sale/tests/test_frontend.py

```diff
@@ -1187,6 +1187,20 @@ def test_res_partner_scan_barcode(self):
         self.main_pos_config.with_user(self.pos_user).open_ui()
         self.start_tour("/pos/ui?config_id=%d" % self.main_pos_config.id, 'BarcodeScanPartnerTour', login="pos_user")
 
+    def test_refund_few_quantities(self):
+        """ Test to check that refund works with quantities of less than 0.5 """
```

Comment:
Is it really for less than 0.5 or just something less than 1.0 ? e.g is 0.9 working without the fix ?

Reply:
For a quantity of at least 0.5, it pass the test without the fix

Reply:
Ok it makes sense, floatIsZero is actually rounding the value before comparing to 0